### ISSUE TITLE
App fixing / clean up

### ISSUE THREAD

**Issue by Pauleing** [SocialHackersClass10/ChalleduApp#87]
* remove useless files (.DS_store, package.lock or yarn.lock outside of app, etc..), complete .gitignore to ignore these type of files permanentely
* Put components in the same folder 
* Make sure all files respect the same naming convention: first letter is uppercase, add a suffix to specify what it is (..Component, ..Provider)
* Fix CSS / design of the app to improve general rendering

**Comment by Jallow9** [comment]
am having this error from the backend when registering

 UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'password' of undefined
and
 UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().


Also in the frontend most of the files complied with warning
./src/Components/AllUsersList.js
  Line 20:6:  React Hook useEffect has a missing dependency: 'user.tokens.access_token'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

**Comment by Pauleing** [comment]
About exhaustive-deps see : 
https://reactjs.org/docs/hooks-reference.html#useeffect

add : 
   // eslint-disable-next-line react-hooks/exhaustive-deps

**Comment by Stef-Lev** [comment]
Because we have made some changes in the directory structure, I suggest we make the final changes and merge this branch when all tickets are complete in order to avoid a havoc of conflicts.

**Comment by Pauleing** [comment]
Yes I agree with @Stef-Lev , there will be more changes this week, so let's keep this open and you will merge master with your branch to add the changes to what you've done, then let's do a final merge
